configure.ac

```diff
@@ -184,6 +184,18 @@ AC_ARG_WITH(
     [with_bundled_protobuf="$withval"],
     [with_bundled_protobuf="detect"]
 )
+AC_ARG_ENABLE(
+    [ml],
+    [AS_HELP_STRING([--enable-ml], [Enable anomaly detection @<:@no@:>@])],
+    [enable_ml="yes"],
+    [enable_ml="no"]
+)
+AC_ARG_ENABLE(
```

Comment:
as an optional module we should probably add it to `netdata -W buildinfo`?

Reply:
Thanks, will do. I completely forgot this.

Reply:
yep - that will be very useful for telemetry too.